packages/translations/src/localizations.ts

```diff
@@ -11,6 +11,11 @@ const shortMonth = (lng: string, date: Date): string =>
     month: 'short',
   })(date);
 
+const longMonth = (lng: string, date: Date): string =>
+intlFormatter(lng, {
```

Comment:
Should be indented so it looks like prettier isn't running.

Reply:
My bad, it should be fixed now 👍🏻